main.ino

```diff
@@ -70,20 +70,29 @@ void loop()
         Serial.println();
     }
     
-    statusCheckCount = (statusCheckCount + 1) % statusCheckInterval;
-    if (statusCheckCount == 0) {
-        statusText = "";
+    unsigned long timeSinceStart = millis();
+    if (!lastStatusCheckTime
+        || timeSinceStart < lastStatusCheckTime // millis() overflow
+        || timeSinceStart - lastStatusCheckTime >= statusCheckIntervalMs) {
+
+        String statusText = "";
+        lastStatusCheckTime = timeSinceStart;
 
         if (!LBattery.isCharging() && LBattery.level() < 30) {
             statusText += "Battery level is less than 30%! "
```

Comment:
It would be nice to send actual level as well.
"ALERT: Your Majesty, it's Battery speaking. My level is X%, that's below 30% threshold. Please charge me or I will die s... #&^%)(#$& CORRUPTED SMS BODY"